src/main/java/com/aws/greengrass/device/session/SessionConfig.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.device.session;
+
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.logging.api.Logger;
+import com.aws.greengrass.logging.impl.LogManager;
+import com.aws.greengrass.util.Coerce;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class SessionConfig {
+    private static final Logger LOGGER = LogManager.getLogger(SessionConfig.class);
+    // arbitrarily set default
+    protected static final int DEFAULT_SESSION_CAPACITY = 100_000;
+    public static final String CLIENT_DEVICE_AUTH_SESSION_CAPACITY_TOPIC = "clientDeviceAuthSessionCapacity";
```

Comment:
remove cda from this string, we know it is cda because customers configure it under the cda key.

Reply:
makes sense, updated.